### ISSUE TITLE
Drow AC and AB penalty

### ISSUE THREAD

**Issue by jd28** [jd28/the_awakening#1]
Glory: I made a drow female today and noticed that the ac/ab penalty did not go away indoors (kua toa caves) and at night. Think this needs to be checked out - pretty sure it does it on the drow male as well. Thanks!

Moved from forums.

**Comment by GuileGenisys** [comment]
Yes, since the subrace special abilities are gone, I think removing the penalties would be fair...
